test/e2e/basic_test.go

```diff
@@ -92,19 +92,28 @@ func checkPodIP(t *testing.T, podNetworkCIDR string, podIP *net.IP) {
 	}
 }
 
-func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName string) {
+func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName string, isWindowsNode bool) {
 	var antreaPodName string
 	var err error
 	if antreaPodName, err = data.getAntreaPodOnNode(nodeName); err != nil {
 		t.Fatalf("Error when retrieving the name of the Antrea Pod running on Node '%s': %v", nodeName, err)
 	}
 	t.Logf("The Antrea Pod for Node '%s' is '%s'", nodeName, antreaPodName)
 
-	cmds := []string{"antctl", "get", "podinterface", podName, "-n", testNamespace, "-o", "json"}
-	stdout, _, err := runAntctl(antreaPodName, cmds, data)
+	var stdout string
+	if isWindowsNode {
+		antctlCmd := fmt.Sprintf("C:/k/antrea/bin/antctl.exe get podinterface %s -n %s -o json", podName, testNamespace)
+		envCmd := fmt.Sprintf("export POD_NAME=antrea-agent;export KUBERNETES_SERVICE_HOST=%s;export KUBERNETES_SERVICE_PORT=%d", clusterInfo.k8sServiceHost, clusterInfo.k8sServicePort)
```

Comment:
Why it needs the K8s service env vars? I thought antctl.exe connects to local endpoint directly?

Reply:
K8s service env vars is not required by this special command itself, but required by antctl.exe . I thinkd it should related with this line https://github.com/antrea-io/antrea/blob/79587c7fb30256cabd7470eaa0e96fde93905d1d/pkg/antctl/client.go#L107

Reply:
I opened an issue a while back about this: https://github.com/antrea-io/antrea/issues/2104